.travis.yml

```diff
@@ -186,12 +186,36 @@ matrix:
         - DESC="build with OpenJDK8"
         - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
 
+    # OpenJDK9 build
+    - jdk: openjdk9
+      env:
+        - DESC="build with OpenJDK9"
+        - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
+
+    # OpenJDK10 build
+    - jdk: openjdk10
+      env:
+        - DESC="build with OpenJDK10"
+        - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
+
+    # OpenJDK11 build
+    - jdk: openjdk11
+      env:
+        - DESC="build with OpenJDK11"
+        - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
```

Comment:
as we start compilation on multiple jdk versions, we need to update headers of non-compilable sources to reference what version of jdk it is compilable to skip compilation on others BUT do compile on required.
ones we upograde to sertain version of jdk as minimal required, all previously non-compiled inputs should be moved to compiled

Reply:
done